docs/spectrum_collection.rst

```diff
@@ -0,0 +1,81 @@
+===================
+Spectrum Collection
+===================
+
```

Comment:
I think this needs a warning about the API not being stable.  I was about to type something here but realized I can just commit directly so will just do that.